docs/source/API/core/parallel-dispatch/parallel_reduce.rst

```diff
@@ -136,7 +136,7 @@ Requirements:
   - ``HandleType`` is an ``IntegerType`` if ``ExecPolicy`` is an ``IntegerType`` else it is ``ExecPolicy::member_type``.
 * If ``ExecPolicy`` is ``MDRangePolicy`` the ``functor`` has a member function of the form ``operator() (const IntegerType& i0, ... , const IntegerType& iN, ReducerValueType& value) const`` or ``operator() (const WorkTag, const IntegerType& i0, ... , const IntegerType& iN, ReducerValueType& value) const``.
 
-  - The ``WorkTag`` free form of the operator is used if ``ExecPolicy::work_tag`` is not ``void``.
+  - The ``WorkTag`` free form of the operator is used if ``ExecPolicy::work_tag`` is ``void``.
```

Comment:
```suggestion
  - If ``ExecPolicy::work_tag`` is ``void``, the overload without a ``WorkTag`` argument is used.
```

Reply:
The suggested change is easier to read for me.